models/vision/classification/alexnet/run_alexnet.sh

```diff
@@ -16,8 +17,15 @@ $curr_dir/../../get_images.sh $image_dir
 if [[ $TEST_WITH_GPU -eq 1 ]]; then
   echo "======== Testing with ODLA TensorRT ========"
   python3 $curr_dir/../../invoke_halo.py --model $model_file --label-file $curr_dir/../1000_labels.txt --image-dir $image_dir --odla tensorrt
+# RUN: FileCheck --input-file /tmp/alexnet_tensorrt.txt %s
```

Comment:
how is "/tmp/alexnet_tensorrt.txt" generated?
should we use something like "RUN: %s | FileCheck %s" ?

Reply:
"/tmp/alexnet_tensorrt.txt" is generated by  invoke_halo.py line 196-205.
FileCheck not work well for shell & python script's screen output

Reply:
I see